R/process_data_set_natalia.R

```diff
@@ -0,0 +1,27 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
```

Comment:
Could you add a line of code indicating the name of the folder in the 538 GitHub? For example, [line 5](https://github.com/rudeboybert/fivethirtyeight/blob/master/data-raw/process_data_sets_maggie.R#L5) of `process_data_sets_maggie.R`
```
# undefeated-boxers---------------------------------------------------------------------
```